teachopencadd/talktorials/001_query_chembl/talktorial.ipynb

```diff
@@ -830,18 +665,18 @@
        "       'nmol/L', \"10'-10M\"], dtype=object)"
       ]
      },
-     "execution_count": 40,
+     "execution_count": 13,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "bioactivities_df.units.unique()"
+    "bioactivities_df[\"units\"].unique()"
```

Comment:
Dominique suggested that we use bioactivities_df["units"] instead of "bioactivities_df.units", so I just change it and the following.